subprojects/base-services/build.gradle.kts

```diff
@@ -14,6 +14,13 @@ tasks.named<JavaCompile>("compileTestJava") {
     options.release.set(8)
 }
 
+/**
+ * Use Java 8 compatibility for JMH benchmarks
+ */
+tasks.named<JavaCompile>("jmhCompileGeneratedClasses") {
+    options.release.set(8)
```

Comment:
Should this go to the `gradlebuild.jmh` plugin? Any idea why it doesn't compile with Spock 2.2?

Reply:
I think there is a problem here with how `usedInWorkers()` works, as it will override the setting set by the JMH plugin. That's why above we also set the target for `compileTestJava` the same way.

Reply:
Fine with me.